lib/govuk/dummy_content_store/content.rb

```diff
@@ -15,6 +16,13 @@ def call(env)
         example = repository.find_by_base_path(env["PATH_INFO"])
         if example
           present_example(example)
+        elsif fallback_url ||= "https://www.gov.uk/api/content"
```

Comment:
Ok, I was trying to configure it with the OptionParser, as
`dummy_content_store -u https://www.gov.uk/api/content` or
`dummy_content_store --url https://www.gov.uk/api/content`